http/headers/Sec-Purpose.json

```diff
@@ -4,17 +4,27 @@
       "Sec-Purpose": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Sec-Purpose",
-          "spec_url": "https://fetch.spec.whatwg.org/#sec-purpose-header",
+          "spec_url": [
+            "https://fetch.spec.whatwg.org/#sec-purpose-header",
+            "https://wicg.github.io/nav-speculation/prefetch.html#sec-purpose-header"
+          ],
           "support": {
             "chrome": {
-              "version_added": false,
-              "notes": "Chrome uses the legacy <code>Purpose: prefetch</code> header to indicate a request is a prefetch. See <a href='https://crbug.com/1358419'>bug 1358419</a>."
+              "version_added": "110",
+              "notes": [
```

Comment:
hm, yeah, complex. Do we want sub features maybe?

- Sec-Purpose
  - speculationrules
  - prefetch

Reply:
You are right, that is pretty horrible and complex. I have tried breaking it up into subfeatures as suggested in the next commit, and I think it is definitely easier to understand now.